pyangbind/plugin/pybind.py

```diff
@@ -1463,12 +1463,14 @@ def get_element(ctx, fd, element, module, parent, path, parent_cfg=True, choice=
                 register_paths=register_paths,
             )
 
+            elemconfig = class_bool_map[element.search_one("config").arg] if element.search_one("config") else True
```

Comment:
As a general practice, I try to avoid duplicated function calls like this, especially in a single line, just to avoid the additional computational overhead. And to improve readability a bit. I think this might read a bit better as
```python
elemconfig = element.search_one("config")
elemconfig = class_bool_map[elemconfig.arg] if elemconfig else True
```

Reply:
Hi @tarkatronic , thanks for the prompt review!

To be honest, I didn't pay a lot of attention to the code itself, I just focused on the desired functionality. That said, I just copied that one-liner from line 1632 of the same file (where YANG leaves are processed). So, now we have the same one-liner in two parts of the same function (get_element)

After your comment I'm thinking I should define a function to implement this functionality instead of duplicating the code. On the other hand, I'm not sure that in this case it would provide any benefits.

What's your take in this?